linera-core/src/node.rs

```diff
@@ -103,6 +103,10 @@ pub trait ValidatorNode {
 
     /// Returns the hash of the `Certificate` that last used a blob.
     async fn blob_last_used_by(&self, blob_id: BlobId) -> Result<CryptoHash, NodeError>;
+
+    /// Returns the hashes of the `Certificates` that last used some blobs.
```

Comment:
AFAIK we use back quotes for actual type names
```suggestion
    /// Returns the hashes of the `Certificate`s that last used some blobs.
```

Reply:
Ok.